python/aixexplainer.Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM python:3.7
+FROM python:3.7-slim
 
 COPY third_party third_party
 
 COPY kserve kserve
 RUN pip install --no-cache-dir --upgrade pip && pip install --no-cache-dir -e ./kserve
 
+RUN apt update && apt install -y build-essential
```

Comment:
`python:3.7-slim` does not include gcc which is used fro installing few pip dependencies in `aixexplainer`.
we install  `build-essential` and remove it after, as no longer needed